crates/attestation/src/crypto/crl.rs

```diff
@@ -88,6 +80,21 @@ impl<'a> PkiPathCRLCheck<'a> for PkiPath<'a> {
                     }
                 }
 
+                if validated_crls.contains(&url) {
+                    // Using the parent cert CRL
+                    // Should only be for AMD.
```

Comment:
Is this something that's specified in the AMD documentation, or just something empirically encountered?
Since in the second case, maybe we shouldn't do this check?

Reply:
Ah, I see that that's because we only ever put in the root CA and one VCEK cert, okay.